app/src/main/java/com/omelan/cofi/MainActivity.kt

```diff
@@ -193,9 +193,12 @@ class MainActivity : MonetCompatActivity() {
 
     @Composable
     fun MainAddRecipe(goBack: () -> Unit, db: AppDatabase) {
+        var isRecipeSave = false
```

Comment:
Check your code for typos before submitting 
```suggestion
        var isRecipeSaved = false
```